test/modules/logicModule/LM_PC_KPIRewarder_v1.t.sol

```diff
@@ -48,7 +48,7 @@ contract LM_PC_KPIRewarder_v1Test is ModuleTest {
 
     // Mock data for assertions
     bytes32 constant MOCK_ASSERTION_DATA_ID = "0x1234";
-    bytes32 constant MOCK_ASSERTION_DATA = "This is test data";
+    uint constant MOCK_ASSERTION_DATA = 100;
```

Comment:
Shouldnt this be named mock asserted Value ?

Reply:
Good call. will update accordingly